addon/helpers/equal.js

```diff
@@ -1,3 +1,13 @@
 export function equalHelper(params) {
-  return params[0] === params[1];
+    let len = params.length;
```

Comment:
@Keeo - Ya, I think `params.every(cb)` would also be fine, but I do generally prefer a simple for loop.